File: Sample/java/src/main/java/net/nend/sample/java/banner/layoutpatterns/TopActivity.java
package net.nend.sample.java.banner.layoutpatterns;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

import net.nend.sample.java.R;

public class TopActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.top);
    }
}

File: Sample/kotlin/src/main/java/net/nend/sample/kotlin/banner/layoutpatterns/TopActivity.kt
package net.nend.sample.kotlin.banner.layoutpatterns

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import net.nend.sample.kotlin.R

class TopActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.top)
    }
}